cmd/main.go

```diff
@@ -126,6 +129,9 @@ func main() {
 
 	cmdRoot.PersistentFlags().StringVarP(&contentTypeFlag, "content-type", "t", "text/plain", "Content type of the payload, may be a MIME type or 'hex'")
 	cmdRoot.PersistentFlags().BoolVar(&insecure, "insecure", false, "Skip TLS validation")
+	cmdRoot.PersistentFlags().StringVarP(&tlsRoute,"tlsPath","T","","Absolute path to cert file")
+	cmdRoot.PersistentFlags().StringVarP(&clientUsername,"clientUsername","U","","hono client username")
```

Comment:
As `clientUsername` and `clientPassword` only work with the client part, I think they should be added under the `cmdConsume` command.

The `tlsPath` parameter seems to be useful for TLS in general, right?

Reply:
I totally agree. See new Commit.